presto-native-execution/presto_cpp/main/PeriodicTaskManager.cpp

```diff
@@ -469,10 +469,11 @@ void PeriodicTaskManager::addWatchdogTask() {
         }
         RECORD_METRIC_VALUE(kCounterNumStuckDrivers, stuckOpCalls.size());
 
-        // Detach worker from the cluster if more than half of driver threads
-        // are blocked by stuck operators (one unique operator can only get
-        // stuck on one unique thread).
-        if (stuckOpCalls.size() > numDriverThreads_ / 2) {
+        // Detach worker from the cluster if more than a certain number of
+        // driver threads are blocked by stuck operators (one unique operator
+        // can only get stuck on one unique thread).
+        if (stuckOpCalls.size() >=
+            SystemConfig::instance()->driverNumStuckOperatorsToDetachWorker()) {
```

Comment:
do you want to take
```
std::max(SystemConfig::instance()->driverNumStuckOperatorsToDetachWorker(), max number of driver threads?);
```

Reply:
@xiaoxmeng 
What do you mean by "max number of driver threads"?

Reply:
yeah, std::min. "max number of driver threads" the number of driver threads